docs/examples/qcrad-consistency-irradiance.py

```diff
@@ -0,0 +1,87 @@
+"""
+QCrad Consistency for Irradiance Data
+=====================================
+
+Check consistency of GHI, DHI and DNI using QCRad criteria.
+"""
+
+# %%
+# Identifying and filtering out invalid irradiance data is a
+# useful way to reduce noise during analysis. In this example,
+# we use :py:func:`pvanalytics.quality.check_irradiance_consistency_qcrad`
+# to check the consistency of GHI, DHI and DNI data using QCRad criteria.
+# For this example we will use the RMIS weather system located on the
+# NREL campus in CO.
+
+
+import pvanalytics
+from pvanalytics.quality.irradiance import check_irradiance_consistency_qcrad
+import pvlib
+import matplotlib.pyplot as plt
+import pandas as pd
+import pathlib
+
+# %%
+# First, read in the RMIS NREL system. This data set contains
+# 5-minute right-aligned sampled data. It includes POA, GHI,
+# DNI, DHI, and GNI measurements.
+
+pvanalytics_dir = pathlib.Path(pvanalytics.__file__).parent
+rmis_file = pvanalytics_dir / 'data' / 'irradiance_RMIS_NREL.csv'
+data = pd.read_csv(rmis_file, index_col=0, parse_dates=True)
+
+# %%
+# Now generate solar zenith measurements for the location,
+# based on the data's time zone and site latitude-longitude
+# coordinates.
+latitude = 39.742
+longitude = -105.18
+time_zone = "Etc/GMT+7"
+data = data.tz_localize(time_zone)
+solar_position = pvlib.solarposition.get_solarposition(data.index,
+                                                       latitude,
+                                                       longitude)
+
+# %%
+# Use
+# :py:func:`pvanalytics.quality.irradiance.check_irradiance_consistency_qcrad`
+# to generate the QCRAD consistency mask.
+
+qcrad_consistency_mask = check_irradiance_consistency_qcrad(
+    ghi=data['irradiance_ghi__7981'],
+    solar_zenith=solar_position['zenith'],
+    dhi=data['irradiance_dhi__7983'],
+    dni=data['irradiance_dni__7982'])
```

Comment:
This parameter ordering is strange to me.  I traced it back to the original from SFA and didn't find any discussion about why `dhi` and `dni` aren't next to `ghi`.  Maybe something to do with consistency with other SFA functions where `dni` and `dhi` are optional?

Note: I'm not saying anything should be changed about this example since it's just following the function's ordering.  More wondering if we should change the parameter ordering in the function itself.

Reply:
Updated the ordering in commit https://github.com/pvlib/pvanalytics/pull/140/commits/2befe871d02ef3e523939fd42bfd70922d4dc862